lib/moderate_parameters/breadcrumbs.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ModerateParameters
+  module Breadcrumbs
+    def [](key)
+      internal_param_logging(key, 'read', caller_locations) if ModerateParameters.breadcrumbs_enabled
+      super
+    end
+
+    def []=(key, value)
+      internal_param_logging(key, 'overwritten', caller_locations) if ModerateParameters.breadcrumbs_enabled
+      super
+    end
+
+    def extract!(*keys)
```

Comment:
Are there plans to add additional methods that are deprecated?

Reply:
Yes